src/pool/srv_metrics.c

```diff
@@ -57,6 +51,36 @@ ds_pool_metrics_alloc(const char *path, int tgt_id)
 	else
 		d_tm_record_timestamp(started);
 
+	rc = d_tm_add_metric(&metrics->handle_eviction, D_TM_COUNTER,
+			     "Total number of evicted pool handles", "hdls",
+			     "%s/evicted_handles", path);
+	if (rc != 0)
+		D_ERROR("Couldn't add pool evicted hdl counter: "DF_RC"\n", DP_RC(rc));
```

Comment:
We have three different error formats that I can see:
- `src/container/srv_metrics.c` 
  - `D_ERROR("failed to add container destroy counter: " DF_RC "\n", DP_RC(rc));`
- `src/object/srv_mod.c`
  - `D_WARN("Failed to create inprogress cnt sensor: "DF_RC"\n", DP_RC(rc));`
- `src/pool/srv_metrics.c`
  - `D_ERROR("Couldn't add pool evicted hdl counter: "DF_RC"\n", DP_RC(rc));`

Should these all be the same? I.e. similar format, wording, ERR/WARN. If so, I can create a ticket since the inconsistency existed before this PR